Data/Queue/Traits.juvix

```diff
@@ -0,0 +1,41 @@
+module Data.Queue.Traits;
```

Comment:
In order to be consistent with the existing codebase, this should be moved to a local module named `QueueTraits` in `Data.Queue.Base`.